README.md

```diff
@@ -145,7 +173,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+main.cpp
```

Comment:
# 3

This code is a correct solution to the wrong problem (which is a shame, because it's a cool idea). 

The question called for you to feed sentences INTO the state machine and see if they matched. Your code generates random sentences based on the grammar.

Also, your generation doesn't really use the state machine properly. It simply iterates through the states, ignoring the next state. As a result, you can only ever get two adjectives out of a run, and they'll always be in the order "lazy" "sme;lly".